api/Selection.json

```diff
@@ -0,0 +1,317 @@
+{
+  "api": {
+    "Selection": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection",
+        "support": {
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "The <a href='https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onselectionchange'><code>GlobalEventHandlers.onselectionchange</code></a> and <a href='https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onselectstart'><code>GlobalEventHandlers.onselectstart</code></a> event handlers are supported as of Firefox 52."
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": true
+          },
+          "webview_android": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true,
+            "notes": "The <a href='https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onselectionchange'><code>GlobalEventHandlers.onselectionchange</code></a> and <a href='https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onselectstart'><code>GlobalEventHandlers.onselectstart</code></a> event handlers are supported as of Firefox 52."
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "modify": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/modify",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
```

Comment:
For this interface, both flavors of Opera should match Chrome. Please check throughout.